### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mattprice-28**
Reinitialised repo and added code to tables.sql to create a new changelog table to record audit data. Added a new file changelogrule.sql to apply the rule to the response table, also added this to the shell script.

### CODE REVIEW

File: load_data.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+log="output-log.txt"
+
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters. Usage: load_data.sh <username> <password>"
+    exit
+else
+    export PGPASSWORD=$2
+fi
+echo "Dropping existing tables..."
+psql --host=localhost --port=5432 --username=$1 --dbname=validationdb -a -f droptables.sql
```
Comment (RyanKilbane) [inline load_data.sh]: Do we really want to do this? Especially after Siva hand cranked all those references for IBM.

Comment (mattprice-28) [reply load_data.sh]: The work siva did was included in the tables.sql file

Comment (mattprice-28) [reply load_data.sh]: Siva is going to send me the sql statements he used to insert these so they can be added

**Comment by dllmorgan** [comment]
Change the audit table name to ResponseHistory
No need for a id column on this audit table
Add Not Null to each column
Use timestamptz instead of timestamp for consistency with existing tables
Add formatting on audit table in tables.sql file (i.e. capitalisation, 4 spaces)
Create an Index on Reference, Period, Survey, QuestionCode
Add rule creation to tables.sql
tables.sql | droptables.sql do not belong in the test-data folder

**Pull request merged**